client/devdocs/design/playground-scope.js

```diff
@@ -112,6 +112,7 @@ export TimeSince from 'components/time-since';
 export Timezone from 'components/timezone';
 export TokenFields from 'components/token-field';
 export Tooltip from 'components/tooltip';
+export UserItem from 'components/user/docs/example';
```

Comment:
I don't think this should be the example here - it should be the actual component.

Reply:
Thanks @scruffian, I'll correct that.

Reply:
Hi @scruffian, I've hopefully addressed this in https://github.com/Automattic/wp-calypso/pull/26406 - I found it problematic getting the component to work in the playground, so removed it from the scope for now.